index.bs

```diff
@@ -672,7 +672,7 @@ Unlike WebGPU, this API does not intrinsically support custom shader authoring;
 
 The WebGPU API identifies <a href="https://gpuweb.github.io/gpuweb/#privacy-machine-artifacts">machine-specific artifacts</a> as a privacy consideration. Similarly, the WebNN API's compute unit scheduling may under certain circumstances introduce a fingerprint. However, similarly to WebGPU, such fingerprints are identical across most or all of the devices of each vendor, mitigating the concern. Furthermore, software implementations can be used to further eliminate such artifacts.
 
-The WebNN API defines two developer-settable preferences to help inform [[#programming-model-device-selection]] and allow the implementation to better select the most appropriate underlying execution device for the workload. [=Device type=] normatively indicates the kind of device and is either {{MLDeviceType/"cpu"}} or {{MLDeviceType/"gpu"}}. If this type cannot be satisfied, an "{{OperationError}}" {{DOMException}} is thrown, thus this type can in some cases add two bits of entropy to the fingerprint. [=Power preference=] indicates preference as related to the power consumption and is considered a hint only and as such does not increase entropy of the fingerprint.
+The WebNN API defines two developer-settable preferences to help inform [[#programming-model-device-selection]] and allow the implementation to better select the most appropriate underlying execution device for the workload. [=Device type=] normatively indicates the kind of device is one of: {{MLDeviceType/"cpu"}}, {{MLDeviceType/"gpu"}}, {{MLDeviceType/"npu"}}. If this type cannot be satisfied, an "{{OperationError}}" {{DOMException}} is thrown, and thus this type can in some cases add two bits of entropy to the fingerprint. [=Power preference=] indicates preference as related to the power consumption and is considered a hint only and as such does not increase entropy of the fingerprint.
```

Comment:
This is based on a quite old (Feb 8) commit, and there's a merge conflict affecting the _"The WebNN API..."_ paragraph. With that fixed locally, my lint tool doesn't detect any errors. 🥳

Reply:
@fdwr can you push an update to resolve this conflict?

And thanks @inexorabletash for checking this with your lint tool. I'd like to test it when you're ready to share it with the world :-)

Reply:
Would it make sense to swap "gpu" and "npu", and allow backwards fallback (from GPU to NPU or CPU, and from NPU to CPU). Then, "gpu" would mean that the implementation can choose the device and the fallback order. AFAICT, this supports the current use cases. 
If we want an error if the given device cannot be used, then we might need to add manual fallback options later.